shell/platform/windows/platform_handler_win32.cc

```diff
@@ -194,30 +202,38 @@ std::unique_ptr<PlatformHandler> PlatformHandler::Create(
   return std::make_unique<PlatformHandlerWin32>(messenger, view);
 }
 
-PlatformHandlerWin32::PlatformHandlerWin32(BinaryMessenger* messenger,
-                                           FlutterWindowsView* view)
-    : PlatformHandler(messenger), view_(view) {}
+PlatformHandlerWin32::PlatformHandlerWin32(
+    BinaryMessenger* messenger,
+    FlutterWindowsView* view,
+    std::optional<ScopedClipboardInterface*> clipboard_reference = nullptr)
+    : PlatformHandler(messenger), view_(view) {
+  if (clipboard_reference == nullptr) {
+    ScopedClipboard clipboard;
+    clipboard_reference = &clipboard;
+  }
```

Comment:
Use of `clipboard` after this line has undefined behavior, because it's destroyed here. If you ran with your PR outside of the context of unit tests, you would almost certainly crash on trying to copy or past anything.

If the goal is for the handler to own the clipboard, then both the member variable and the constructor argument need to be `unique_ptr`s.

Reply:
Thanks, good catch.  I think this does it: https://github.com/flutter/engine/pull/32038/commits/cad4781b1a6410b431c2258cde2a2322188c9d8a

I tried running an app after that change and messing around with the clipboard and wasn't able to cause a crash.